docs/source/receive-adapter.md

```diff
@@ -0,0 +1,156 @@
+# Receive adapter
+
+A _receive adapter_ is a data-plane component which either
+
+- produces [CloudEvents](https://github.com/cloudevents/spec),
```

Comment:
commas aren't needed in bullet point lists

Reply:
fixed